### ISSUE TITLE
Test result in a real scenario

### ISSUE THREAD

**Issue by pandysong** [FrancoisGrondin/steernet#2]
If I understand correctly, the sample clip is generated uinsg librispeech. 

I wander if there are test result on a real recording clip?

**Comment by FrancoisGrondin** [comment]
Yes exactly. We don't have results on real recordings. We would ideally need a training set generated in a real environment and a corresponding test set. Do you have any in mind?

**Comment by pandysong** [comment]
I have no such recording for training. Data argumentation might be good enough to add noise or reverberation on to the clip?

But before that I am curious about the performance of the current available model on a real recording (which I do not have as well)

Thanks for your update.

**Comment by FrancoisGrondin** [comment]
I see. For now I don't have any test data recorded in real conditions. This would be something to do in the future.

**Comment by pandysong** [comment]
If I could capture recording data for a mic array, e.g. 4 Mic in a circular pattern. Would it help?

**Comment by FrancoisGrondin** [comment]
Sure. For now the network has been trained to perform on 2 speech sources, so you could record two sources at different direction of arrival. Thanks!
